datajoint/fetch.py

```diff
@@ -37,9 +38,15 @@ def _initialize_behavior(self):
     @property
     def squeeze(self):
         """
+        DEPRECATED
```

Comment:
Let's think about it but let's not all of the sudden go about changing everything.